Index: qdigidoc/breakpad/CMakeLists.txt
===================================================================
--- qdigidoc/breakpad/CMakeLists.txt	(revision 112)
+++ qdigidoc/breakpad/CMakeLists.txt	(working copy)
@@ -34,7 +34,9 @@
 	add_definitions( -D_FILE_OFFSET_BITS=64 )
 	set( COMMON_FLAGS "-fno-exceptions -Wno-unused-parameter -Wno-missing-field-initializers" )
 	set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${COMMON_FLAGS}" )
-	set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${COMMON_FLAGS} -fno-rtti -fno-threadsafe-statics" )
+        # See https://code.google.com/p/google-breakpad/issues/detail?id=481#c5
+        set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${COMMON_FLAGS} -fno-rtti -fno-threadsafe-statics -std=gnu++0x" )
+	        
 if( APPLE )
 	set( SOURCES ${SOURCES}
 		${BREAKPAD_PATH}/client/mac/handler/exception_handler.cc
