diff -urNp -x '*.orig' qdigidoc-3.13.6.org/client/qdigidoc-client.desktop qdigidoc-3.13.6/client/qdigidoc-client.desktop
--- qdigidoc-3.13.6.org/client/qdigidoc-client.desktop	2018-04-06 13:46:51.000000000 +0200
+++ qdigidoc-3.13.6/client/qdigidoc-client.desktop	2021-10-04 09:28:10.407102895 +0200
@@ -9,5 +9,5 @@ Name=DigiDoc Client
 Name[et]=DigiDoc klient
 Name[ru]=DigiDoc клиент
 
-Categories=Qt;Office;
+Categories=Qt;Office;Security;Utility;
 MimeType=application/vnd.etsi.asic-e+zip;application/vnd.etsi.asic-s+zip;application/x-ddoc;application/vnd.lt.archyvai.adoc-2008;application/x-p12d;
diff -urNp -x '*.orig' qdigidoc-3.13.6.org/crypto/qdigidoc-crypto.desktop qdigidoc-3.13.6/crypto/qdigidoc-crypto.desktop
--- qdigidoc-3.13.6.org/crypto/qdigidoc-crypto.desktop	2018-04-06 13:46:51.000000000 +0200
+++ qdigidoc-3.13.6/crypto/qdigidoc-crypto.desktop	2021-10-04 09:28:10.407102895 +0200
@@ -12,5 +12,5 @@ Name[ru]=DigiDoc Crypto
 Keywords=Crypto;Digital;Signature;
 Keywords[et_EE]=Krüpto;Digitaalne;Allkiri;
 
-Categories=Qt;Office;
+Categories=Qt;Office;Security;Utility;
 MimeType=application/x-cdoc;
